invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,3 +29,24 @@ def create_demo_record(data):
     service = current_rdm_records.records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task(ignore_result=True)
+def update_expired_embargos():
```

Comment:
sorry I just realized, should this be moved to `services/tasks.py` (create if not existing)... it doesnt have any relation to fixtures, right?